code/__HELPERS/_lists.dm

```diff
@@ -258,10 +415,18 @@
 //Removes any null entries from the list
 //Returns TRUE if the list had nulls, FALSE otherwise
 /proc/listclearnulls(list/L)
-	var/start_len = L.len
-	var/list/N = new(start_len)
-	L -= N
-	return L.len < start_len
+	return (list_to_clear.RemoveAll(null) > 0)
```

Comment:
These two lines have a different name for the list variable. `L` and `list_to_clear`